<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerDialog</class>
 <widget class="QDialog" name="TimerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>164</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progressBarMicroBreak">
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMicroBreakTimer">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0:60</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBarRestBreak">
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRestBreakTimer">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0:60</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QProgressBar" name="progressBarIdle">
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelIdleTimer">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0:60</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
